### ISSUE TITLE
RC Test: Resistance in a Wire 1.4.0-rc.1

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#66]
@ariel-phet @emily-phet @phet-steele @zepumph, Resistance in a Wire 1.4.0-rc.1 is ready for RC testing. This sim has identical SHAs to 1.3-phetio, but keyboard navigation is enabled. This deviation seems enough to not synchronize minor version numbers. But still using same "stable" shas from that version.

This simulation includes keyboard navigation and those features need to be tested as part of this RC. For expected keyboard interaction behavior, please see the "Keyboard Help Dialog", found in the navigation bar.

**[Link to sim](https://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.4.0-rc.1/resistance-in-a-wire_en.html)**

**[Link to iFrame](https://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.4.0-rc.1/resistance-in-a-wire_en-iframe.html)**

**[Test Matrix](https://docs.google.com/spreadsheets/d/1qNs8hbWvo19YKgn1ds_Y1AuV8N8k9tneGWaRB9N4plE/edit#gid=309825344)**

**Issues to Verify**
As a sanity check, please verify that this issue is fixed in this version:
- [x] https://github.com/phetsims/resistance-in-a-wire/issues/108

**Please also verify**
- [x] stringTest=double (all strings doubled)
- [x] stringTest=long (exceptionally long strings)
- [x] stringTest=X (short strings)
- [x] stringTest=rtl (right-to-left)
- [x] stringTest=xss (test passes if sim does not redirect, OK if sim crashes or fails to fully start)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test
- [x] Screenshot test

If any new issues are found, please note them in https://github.com/phetsims/resistance-in-a-wire/issues and reference this issue.

Please let me know if we should discuss in more detail how to test keyboard navigation features.

@ariel-phet could you please prioritize?

**Comment by jessegreenberg** [comment]
Once testing is complete, we will need to update https://github.com/phetsims/resistance-in-a-wire/issues/114.

**Comment by ariel-phet** [comment]
@phet-steele similar priority to #64

**Comment by brroberts1231** [comment]
@jessegreenberg the link to the testing matrix is giving me a 404 error

**Comment by jessegreenberg** [comment]
Sorry @brroberts1231, I meant to remove that link entirely, I don't have permissions to create a matrix. If you have access could you please create a matrix for this test?  Otherwise, @phet-steele can you please create a matrix?

**Comment by phet-steele** [comment]
> Otherwise, @phet-steele can you please create a matrix?

After #64

**Comment by phet-steele** [comment]
Matrix is linked above.

**Comment by phet-steele** [comment]
@jessegreenberg everything is fine here other than https://github.com/phetsims/john-travoltage/issues/252 and https://github.com/phetsims/balloons-and-static-electricity/issues/320

**Comment by jessegreenberg** [comment]
Thanks @phet-steele!  Commits have been cherry picked for phetsims/john-travoltage#252 and phetsims/molecules-and-light#160. Once I know the status of phetsims/balloons-and-static-electricity#320, we will  be ready to spot check phetsims/john-travoltage#252.

**Comment by jessegreenberg** [comment]
Closing this issue, thanks!
